capacity_sustained_load_harness_test.go

```
@@ -0,0 +1,710 @@
+			fields := strings.Fields(scanner.Text())
+			if len(fields) >= 2 && fields[0] == "VmRSS:" {
+				kilobytes, err := strconv.ParseInt(fields[1], 10, 64)
+				if err != nil || kilobytes <= 0 || kilobytes > int64(^uint64(0)>>1)/1024 {
+					return 0, fmt.Errorf("invalid resident-memory sample")
+				}
+				return kilobytes * 1024, nil
+			}
+		}
+		return 0, fmt.Errorf("resident-memory sample is missing")
+	}
+	output, err := exec.Command("ps", "-o", "rss=", "-p", strconv.Itoa(pid)).Output()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)